src/Scss/variables.scss

```diff
@@ -113,4 +113,19 @@ $white:#ffff;
   cursor: pointer;
   outline: none;
   @extend %drop-shodow;
+}
+
+%make-post-button{
+  grid-row: 5;
+  margin-top: 2em;
+  height: 1.5em;
+  border-radius: 0.5em;
+  color:$white;
+  font-weight: bold;
+  font-size: 1.5em;
+  background-color: $application-blue;
+  @extend %make-post-margin;
+  outline: none;
+  @extend %drop-shodow;
+  margin-bottom: 1em;
```

Comment:
Changing anything in here might change the buttons across the whole document what is this all connected to?